stream/src/placeholder/instance.rs

```diff
@@ -150,20 +150,14 @@ impl<C: Scheme, Si: Sink, St: Stream> Instance<C, Si, St> {
             Sender {
                 cipher: self.cipher.clone(),
                 sink: self.sink,
-
                 max_message_size: self.config.max_message_size,
-                dialer: self.dialer,
-                iter: 0,
-                seq: 0,
+                nonce: nonce::Info::new(self.dialer),
             },
             Receiver {
                 cipher: self.cipher,
                 stream: self.stream,
-
                 max_message_size: self.config.max_message_size,
-                dialer: self.dialer,
-                iter: 0,
-                seq: 0,
+                nonce: nonce::Info::new(!self.dialer),
```

Comment:
This should not be flipped. The nonce used for the `Sender/Receiver` isn't an inverse (it depends on `dialer`/`non-dialer`).